eventuate-messaging-kafka-consumer/src/main/java/io/eventuate/messaging/kafka/consumer/ReactiveKafkaMessageHandler.java

```diff
@@ -0,0 +1,8 @@
+package io.eventuate.messaging.kafka.consumer;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+public interface ReactiveKafkaMessageHandler extends Function<KafkaMessage, CompletableFuture<Void>> {
```

Comment:
This can just be called `KafkaMessageHandler`.

Reply:
Chris, for backward compatibility I kept original KafkaMessageHandler

Reply:
ok